### PR TITLE
Update UG for view, delete, exit

### PR DESCRIPTION
**Pull request by wish2023**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
+`view TASK-ID`
+
+##### Parameters
+The `view` command requires 1 parameter only.
+
+* `TASK_ID` refers to the number shown beside the task that the user wants to delete in the displayed task list. It must be a non-negative integer.
+
+##### Example Usage
+
+Firstly, use [list command](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be edited.
```
Comment (daniellimws) [inline docs/UserGuide.md]: > use list command

"use the list command" instead?

> task to be edited?

to be viewed?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
+The `delete` command requires 1 parameter only.
+
+* `TASK_ID` refers to the number shown beside the task that the user wants to delete in the displayed task list. 
+It must be a non-negative integer.
+
+##### Example Usage
+
+The following shows the output from ATHENA after entering `delete 1`.
+You should expect to see a message to confirm that the task is deleted. 
+ATHENA also prints a message for the user to type in case the task has been deleted accidently.
```
Comment (daniellimws) [inline docs/UserGuide.md]: > prints a message for the user to type

how about "prints the command to restore the task"

> accidently

accidentally?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
-* `v 0`
+##### Command Format
 
-#### Expected output
+`view TASK-ID`
+
+##### Parameters
+The `view` command requires 1 parameter only.
+
+* `TASK_ID` refers to the number shown beside the task that the user wants to delete in the displayed task list. It must be a non-negative integer.
```
Comment (daniellimws) [inline docs/UserGuide.md]: > wants to delete

view?

**Review by daniellimws — COMMENTED** [review]
Some typos

File: docs/UserGuide.md [hunk 4]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
+##### Command Format
 
-* `delete 1`
-* `dl 1`
+`delete TASK-ID`
+
+##### Parameters
+The `delete` command requires 1 parameter only.
+
+* `TASK_ID` refers to the number shown beside the task that the user wants to delete in the displayed task list.
```
Comment (amanda-chua) [inline docs/UserGuide.md]: `TASK-ID` instead of `TASK_ID`

File: docs/UserGuide.md [hunk 5]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
-* `v 0`
+##### Command Format
 
-#### Expected output
+`view TASK-ID`
+
+##### Parameters
+The `view` command requires 1 parameter only.
+
+* `TASK_ID` refers to the number shown beside the task that the user wants to view in the displayed task list. It must be a non-negative integer.
```
Comment (amanda-chua) [inline docs/UserGuide.md]: TASK-ID instead of TASK_ID

File: docs/UserGuide.md [hunk 6]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
+
+##### Example Usage
+
+Firstly, use the [list command](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be viewed.
+The following shows the output from ATHENA after entering `view 0`.
+You should expect to see the details of your task with `TASK_ID` of `0`.
```
Comment (amanda-chua) [inline docs/UserGuide.md]: TASK-ID instead of TASK_ID

**Review by amanda-chua — APPROVED** [review]
Looks good with minor mistakes in naming

File: docs/UserGuide.md [hunk 7]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
+##### Command Format
 
-* `delete 1`
-* `dl 1`
+`delete TASK-ID`
+
+##### Parameters
+The `delete` command requires 1 parameter only.
+
+* `TASK-ID` refers to the number shown beside the task that the user wants to delete in the displayed task list.
```
Comment (alstontham) [inline docs/UserGuide.md:413]: maybe you could add a note here mentioning that the user can view the task number by using the "list" command

File: docs/UserGuide.md [hunk 8]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
-* `v 0`
+##### Command Format
 
-#### Expected output
+`view TASK-ID`
+
+##### Parameters
+The `view` command requires 1 parameter only.
+
+* `TASK-ID` refers to the number shown beside the task that the user wants to view in the displayed task list. It must be a non-negative integer.
```
Comment (alstontham) [inline docs/UserGuide.md:442]: Same comment as for the delete command. Also, would be great to put "It must be a non-negative integer" on a separate line to remain consistent (refer to line 414 for the delete command).

File: docs/UserGuide.md [hunk 9]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
+
+##### Example Usage
+
+Firstly, use the [list command](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be viewed.
```
Comment (alstontham) [inline docs/UserGuide.md:446]: Nice! Perhaps you could include this line under "Example Usage" of the delete command also

File: docs/UserGuide.md [hunk 10]
```diff
@@ -401,34 +401,71 @@ You should expect to see a message to confirm that the time of task with `TASK-I
...
+The following commands on the left will be expanded to the corresponding commands on the right.
+
+| Shortcut | Expanded command |
+|----------|------------------|
+| `v`     | `view`  |
+
+##### Example usage
+
+Firstly, use [list command](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be edited.
+
```
Comment (alstontham) [inline docs/UserGuide.md:466]: Would be good to keep the spacing consistent in each of the "Example usage" sections

**Review by alstontham — CHANGES_REQUESTED** [review]
Looks great! Just some suggestions for keeping each section standardized and consistent with the other sections.

**Pull request merged**